### DISCUSSION TITLE
Cannot use

### DISCUSSION THREAD

**Discussion by wb-eng** [getlantern/lantern-forum-cn#971] [General]
cannot open any website. It's almost the same situation in Android and windows, desktop even worse I think. This has lasted almost 3months, I ask to fix it and prolong my subscribion accordingly. From the usage experience recently, I would like to ask for a 3-months prolong currently and the problem solved, if not, should be lonver.

My account : wixbao@gmail.com

**Comment by MaxMKMax** [comment]
@wb-eng Hello, sorry for the inconvience caused. Would you mind to help us submit an in-app report/ LOG? Then provide me your 8 digit LOG number, i will help you submit your issues to tech team once i receive your log. They can help to solve the issues. Thanks.

**Reply by wb-eng** [reply]
The log uploaded from both Android and Windows with my account wixbao@gmail.com.
Cannot get the CODE in the add device page. If the PRO account already login, in add device page ask you to input the code. If login off my account, cannot open the page. Crazy....

**Reply by wb-eng** [reply]
Reinstalled the app and authorized, the code is 783018. Maybe you can use this for trouble shooting.

**Reply by MaxMKMax** [reply]
@wb-eng Okay noted. I will help to report to tech team and open a ticket to solve the bugs/ issues as soon as possible. Thanks for letting us know, sorry for the inconvenience caused. Truly apologize on that. Thank you so much for being patiently communicate with us.

**Reply by wb-eng** [reply]
Is there any progress? I cnanot use it on my win laptop as well.

**Comment by MaxMKMax** [comment]
Hi @wb-eng I had reported to the tech team last week during the meeting. The case is escalated and they are working on it to fix the issues. Let me recheck the issues with you again, if any new updates i will let you know. 

By the way, can you also provide me some screenshots on the issues you facing? such as the cannot use page and others. It will be helpful to solve the issues.

Sorry for the inconvenience caused and Thank you for supporting Lantern.

**Reply by wb-eng** [reply]
The root cause seems to be the bad connection to server, it switches between different servers. Even cannot open Google. I took a screen recording, but is there any way to upload, size of it is 30M. Hopefully, the upload path can access without proxy.

**Comment by wb-eng** [comment]
Hi @Mike777Khoo , sync with your side what I have done. I switch the default browser from Edge to chrome, open the lantern will jump to the landpage of lantern in chrome. It works then.  I reset the setting of Edge, it cannot work as before what I reported. So I think your side may have a check if any compatiblity issue on edge. The version I used is the latest I think. 
![image](https://github.com/user-attachments/assets/dbc8f26b-cb5e-4c7d-9030-e38549bf37af)

**Reply by wb-eng** [reply]
But anyway, I'd like to say that even in Chrome, it is not as stable as before.
